odtools/odcomparepointcloud/CMakeLists.txt

```diff
@@ -0,0 +1,99 @@
+# odcomparepointcloud - Tool for comparing data between shared point cloud and 
```

Comment:
Yes. Now it is in odtools. Since I am going to AstaZero after lunch for the data collection, it is more important to get the new Docker image versions ready. After that I will try to update odcomparepointcloud by considering timestamps and the two types of SPC.